tests/testdata/default_config.yaml

```diff
@@ -16,6 +16,9 @@ chat:
   # Model to be used for chatting with.
   # Default: /cache/instructlab/models/granite-7b-lab-Q4_K_M.gguf
   model: /cache/instructlab/models/granite-7b-lab-Q4_K_M.gguf
+  # The RAG chat configuration
+  # Default: {}
+  rag: {}
```

Comment:
yeah, this should have some sane defaults or else users will not know how to use this